index.js

```diff
@@ -4,31 +4,51 @@ if (process.platform === 'win32') {
   spawn = require('cross-spawn')
 }
 
-module.exports = function (program, args, cb) {
-  var arrayIndex = arguments.length
-
-  if (typeof args === 'function') {
-    cb = args
-    args = undefined
+/**
+ * Normalizes the arguments passed to `foregroundChild`.
+ *
+ * See the signature of `foregroundChild` for the supported arguments.
+ *
+ * @param fgArgs Array of arguments passed to `foregroundChild`.
+ * @return Normalized arguments
+ * @internal
+ */
+function normalizeFgArgs(fgArgs) {
+  var program, args, cb;
+  var processArgsEnd = fgArgs.length;
+  var lastFgArg = fgArgs[fgArgs.length - 1];
+  if (typeof lastFgArg === "function") {
+    cb = lastFgArg;
+    processArgsEnd--;
```

Comment:
Prefer use of `-= 1` instead for explicitness.

Reply:
Should I push force the update or use extra commit for this fix?